tools/src/test/resources/1.6/invalid-component-ref-1.6.xml

```diff
@@ -10,6 +10,10 @@
                     <name>acme-library</name>
                     <version>1.0.0</version>
                 </component>
+                <component type="library" bom-ref="123">
```

Comment:
this was the misisng cae of duplicate bom-ref.
this one was added to the JOSN example (see https://github.com/CycloneDX/specification/pull/423/files#r1558896105) whene it was unaffective.
Actually, this case is affective in the XML implementation, so it was added in here.